src/pages/rewards/[address]/index.tsx

```diff
@@ -121,11 +168,36 @@ const Rewards: React.FC = () => {
       rewardTokenSymbol,
     };
   }, [farm, provider]);
+
   const [lookup, refetchLookup] = useAsyncState(null, lookupCall);
   const refresh = () => {
     refetchLookup();
   };
 
+  const checkOwner = React.useCallback(async () => {
+    const signer = await getConnectedSigner();
+    if (signer._address == lookup?.owner) {
```

Comment:
nit: Can just return signer._address === lookup?.owner.

Also use `===` and not `==`